src/config/index.ts

```diff
@@ -31,6 +29,15 @@ export const REFERENCE = "OverseasEntitiesReference";
 export const PAYMENT_REQUIRED_HEADER = "x-payment-required";
 export const PAYMENT_PAID = "paid";
 
+//  FEATURE FLAGS
+const envFlagSfx = 'FEATURE_FLAG_';
+enum FeatureFlag {
```

Comment:
This enum is not needed, feature flags are short lived, they will be removed probably in the next two sprints. Please keep things simple

Reply:
Feature Flag will we always, just some will be removed and some will be added

Reply:
Revert this part, please

Reply:
I revert all related existing feature flags
Keep this enum for future feature flags. This is more strict approach, that using now. FeatureFlag enum can be empty.

Reply:
reverted

Reply:
This now looks ok to me.

Reply:
Do not know why the feature flag does no have the date but it is just minor change. Fine to me